x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -111,6 +111,7 @@ export const isolationRequestHandler = function (
   SecuritySolutionRequestHandlerContext
 > {
   return async (context, req, res) => {
+    endpointContext.service.getFeatureUsageService().notifyUsage('HOST_ISOLATION');
```

Comment:
this interface feels a little weird? You call it with a `String`, but it's really a keyword const from `FEATURES`. Maybe this is just due to ts typing, but I would expect actually using an imported constant here, not a string. It's less obvious from the calling site what's going on (but is immediately clear in the service's definition file).

Reply:
@pzl ,
FYI: the string is actually not arbitrary - it has to be a key of the object where @joeypoon defined the feature set. Take a look at the method signature further below.

Reply:
definitely leaning on typescript a bit here. since it's union string typed, intellisense picks it up and we still get autocomplete without needing an imported constant. if we feel strongly about it, I can create another export const/enum to use.